src/models/calendar.rs

```diff
@@ -258,6 +259,7 @@ impl Calendar {
                             originating_goal_id: budget_id.clone(),
                             participating_goals: descendants_added,
                             time_budgets: get_time_budgets_from(self, goal),
+                            time_filters: goal.filters.clone().unwrap(),
```

Comment:
remove duplication

Reply:
see above

Reply:
@thinkrapido Please explain what the duplication is.  
`time_budgets` is a completely different thing from `time_filters`.